source/client/options_impl.cc

```diff
@@ -75,6 +76,14 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
           "--protocol is used. Mutually exclusive with --h2.",
           absl::AsciiStrToLower(nighthawk::client::Protocol_ProtocolOptions_Name(protocol_))),
       false, "", &protocols_allowed, cmd);
+  TCLAP::ValueArg<std::string> http3_protocol_options(
+      "", "http3-protocol-options",
+      "HTTP3 protocol options in json. If specified, Nighthawk uses these HTTP3 protocol options "
+      "when sending requests. Only valid with --protocol http3. Exclusive with any other command "
```

Comment:
nit: Exclusive -> Mutually exclusive to be consistent with the rest of the file.

Reply:
Done in 597fa383b1bc83996aa175548cb6b1de9713acbf.